packages/serverpod_test/lib/serverpod_test.dart

```diff
@@ -0,0 +1,3 @@
+library serverpod_test;
+
+export 'src/serverpod_test.dart';
```

Comment:
Question: Is it possible to split it up a bit to make this project a bit simpler to navigate?

Reply:
Good call out! I should definitely break this file up.